bin/ci/az-publish.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+# $1 => SOURCE PATH
+# $2 => TARGET NAME
+function azupload {
+  az storage blob upload \
+    --container-name "${3}" \
+    --file "${1}" \
+    --name "${2}"
+  echo "${1} artifact uploaded with name ${2}"
+}
+
+function info {
+  if [ "${TERM}" = "dumb" ]; then
+    echo "INFO: $1"
+  else
+    echo "$(tput setaf 2)INFO: $1$(tput sgr0)"
+  fi
+}
+
+function zip_with_ditto {
+  xcrun ditto \
+  -ck --rsrc --sequesterRsrc --keepParent \
+  "${1}" \
+  "${2}"
+  info "Installed ${2}"
+}
+
+if [ -e ./.azure-credentials ]; then
+  source ./.azure-credentials
+fi
+
+if [[ -z "${AZURE_STORAGE_ACCOUNT}" ]]; then
+  echo "AZURE_STORAGE_ACCOUNT is required"
+  exit 1
+fi
+
+if [[ -z "${AZURE_STORAGE_KEY}" ]]; then
+  echo "AZURE_STORAGE_KEY is required"
+  exit 1
+fi
+
+if [[ -z "${AZURE_STORAGE_CONNECTION_STRING}" ]]; then
+  echo "AZURE_STORAGE_CONNECTION_STRING is required"
+  exit 1
+fi
+
+if [ "${BUILD_SOURCESDIRECTORY}" != "" ]; then
+  WORKING_DIR="${BUILD_SOURCESDIRECTORY}"
+else
+  WORKING_DIR="."
+fi
+
+APP_PRODUCT_DIR="${WORKING_DIR}/CalSmokeApp/Products/app/CalSmoke-cal"
+IPA_PRODUCT_DIR="${WORKING_DIR}/CalSmokeApp/Products/ipa/CalSmoke-cal"
+INFO_PLIST="${APP_PRODUCT_DIR}/CalSmoke-cal.app/Info.plist"
+
+# Evaluate CalSmokeApp version (from Info.plist)
+VERSION=$(/usr/libexec/PlistBuddy -c "Print :CFBundleShortVersionString" ${INFO_PLIST})
+
+# Evaluate Xcode version (from Info.plist)
+XC_VERSION=$(/usr/libexec/PlistBuddy -c "Print :DTXcode" ${INFO_PLIST})
+
+# Evaluate git-sha value
+GIT_SHA=$(git rev-parse --verify HEAD | tr -d '\n')
+
```

Comment:
Could you please add this code here?
```
GIT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
# We don't need to use AdHoc when executing locally
if [[ "${GIT_BRANCH}" =~ "tag/" && -e ./.azure-credentials ]]; then
  BUILD_ID="CalWebView-${VERSION}-Xcode-${XC_VERSION}-${GIT_SHA}"
else
  BUILD_ID="CalWebView-${VERSION}-Xcode-${XC_VERSION}-${GIT_SHA}-AdHoc"
fi
```

After that we can use `$BUILD_ID` variable in order to construct names of artifacts

Reply:
Done